runtime/doc/builtin.txt

```diff
@@ -9787,6 +9787,8 @@ setqflist({list} [, {action} [, {what}]])		*setqflist()*
 				current entry.  See |quickfix-index|
 		    items	list of quickfix entries. Same as the {list}
 				argument.
+		    keep_idx	select the entry nearest the currently
+		    		selected entry
```

Comment:
I don't have much experience writing these. I fear this doesn't make a lot of sense. Feedback appreciated. Maybe I should say something like "useful when replacing the quickfix list and you want to keep your position in the list"?

Reply:
Marking this resolved. I think the updated docs for the new `'u'` action are ok. Feedback appreciated!